lib/mongo/server_selector/nearest.rb

```diff
@@ -41,16 +41,25 @@ def name
         :nearest
       end
 
-      # Whether the slaveOk bit should be set on wire protocol messages.
+      # Whether the secondaryOk bit should be set on wire protocol messages.
       #   I.e. whether the operation can be performed on a secondary server.
       #
       # @return [ true ] true
       #
-      # @since 2.0.0
-      def slave_ok?
+      # @since 2.15.0
```

Comment:
We stopped using @since tags because they fail to account for signature expansions/added options, so whenever a method is modified the @since tag may be dropped if the signature changes and for new methods we don't need them.

Reply:
Understood, thanks.